Source/Core/VideoBackends/Software/SWmain.cpp

```diff
@@ -41,7 +41,7 @@
 
 #define VSYNC_ENABLED 0
 
-static volatile u32 s_swapRequested = false;
+static std::atomic<bool> s_swapRequested;
 
 static volatile struct
```

Comment:
Not sure if theres a particular reason why this one needs to be marked as ```volatile```. Ping phire about it?